ORBIT/phases/design/electrical_export.py

```diff
@@ -255,8 +274,6 @@ def compute_number_cables(self):
         num_redundant : int
         """
 
-        _num_redundant = self._design.get("num_redundant", 0)
-
         num_required = np.ceil(self._plant_capacity / self.cable.cable_power)
         num_redundant = self._design.get("num_redundant", 0)
```

Comment:
You mean like how we're treating the get costs? I figured this could be a default_design.yaml in the future and we go through all these design default settings and put in a central location. But I wanted to focus on costs alone with this PR.

Reply:
punting this until a later date.